.github/workflows/maven-verify.yml

```diff
@@ -20,54 +20,54 @@ name: Verify
 on:
   workflow_call:
     inputs:
-      maven_args:
+      maven-args:
```

Comment:
is this backward compatible?

Reply:
not - description update
so I created next branch - version v3

changed to be consequently in options name format